code/game/objects/items/devices/scanners.dm

```diff
@@ -411,7 +411,7 @@ GENE SCANNER
 			render_list += "<span class='notice ml-1'>Detected cybernetic modifications:</span>\n"
 			render_list += "<span class='notice ml-2'>[cyberimp_detect]</span>\n"
 
-	to_chat(user, jointext(render_list, ""), trailing_newline = FALSE) // we handled the last <br> so we don't need handholding
+	to_chat(user, jointext(render_list, ""), trailing_newline = FALSE, type=MESSAGE_TYPE_INFO) // we handled the last <br> so we don't need handholding
```

Comment:
I'd prefer spaces around the = here, since I don't think? the line count is going to hell, and it makes things a bit easier to read

Reply:
could also move the comment to the line before to make things less disastrously long

Reply:
I think it's better now